src/data/Data.js

```diff
@@ -147,7 +147,11 @@ Data.prototype = {
 
 		// Remove the expando if there's no more data
 		if ( key === undefined || jQuery.isEmptyObject( cache ) ) {
-			delete owner[ this.expando ];
+			// Webkit & Blink performance suffers when deleting properties
```

Comment:
I guess `Support: Chrome <=35 - 45` then?  It could have been an issue earlier but it looks like it was [originally reported for Chrome35](https://groups.google.com/a/chromium.org/forum/#!msg/chromium-bugs/2i6fYpkjiQ8/kFxJoAMbFC8J), and it seems to still be an issue last time I tested in 45.

I'm not sure about Safari though. I don't recall ever seeing any webkit/safari bugs for this. I think the "Webkit &" was because it was noticeable in jsperf (https://github.com/jquery/jquery/pull/1664#issuecomment-56916618).